docs/repros/fork-loop.md

```
@@ -35,6 +35,26 @@ scripts/repro-fork.sh
 scripts/repro-fork.sh --runs 20   # local soak — fails fast on first hang
 ```
 
+### Build-only variant
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)